### ISSUE TITLE
Simplify the game? remove power ups, fewer levels

### ISSUE THREAD

**Issue by city41** [city41/breakouts#68]
I've been thinking about this and I wonder if it would actually be better to remove the extra ball and paddle shrinking power ups. Or at least one of them.

Same with the four levels, just have two levels.

And also ditch the who "prerendered vs tile" bg notion that is floating around a bit here and there.

Why? they add complexity to the game without really any benefit. No one actually plays these breakouts. They should be just complicated enough to show a real world-ish use of the engine and nothing more.

Any opinions?

**Comment by ellisonleao** [comment]
Very good point @city41 . I am having some issues on #64 because of some game logic and i think that simplifying on one or two levels and just leaving the power down object could be the best case.

**Comment by city41** [comment]
ok, let's do it.

here's a spreadsheet capturing the state of everything, it's hard to manage 12 engines :)

https://docs.google.com/spreadsheets/d/1PHbn10NqJm4cvzGgPyBS066mDB9-Dc5_D47jb6OaPKs/edit#gid=0

**Comment by jerle76** [comment]
Again, I'll be more than happy to simplify the CreateJS version.

**Comment by jerle76** [comment]
@city41, can you confirm all of the following is to throw away?
- levels 3 and up,
- power up/multi balls,
- prerendered background

**Comment by city41** [comment]
I say we eliminate:

* multiball powerup
* paddle shortening powerdown
* level 3 and 4

as for backgrounds, whichever is easiest for the engine, whether that is tiled or prerendered. The first 3 breakout implementations (Impact, Crafty and Lime) all had the ability to switch between tiled and prerendered with the use of a URL parameter. That was my typical over thinking things :)

**Comment by ellisonleao** [comment]
hm, for melonjs i was keeping the powerdown logic. Should i remove it?

**Comment by city41** [comment]
I think both should go, unless you guys feel strongly otherwise. If we kept one, we might as well keep both of them.

**Comment by jerle76** [comment]
They don't contribute to illustrating the framework's features. They only contribute to the gameplay which isn't the point of this project. I agree they should both go.

**Comment by ellisonleao** [comment]
@city41 i can remove it , no problem :smile:

**Comment by jerle76** [comment]
for the background and without having looked into it seriously, I would lean toward pre-rendered background on account of already having other examples of sprite manipulations in the game (paddle, bricks and ball).

**Comment by city41** [comment]
I think most engines will prefer prerendered backgrounds. But Impact is a good example where tiles are actually more natural. I think any engine can choose to use whichever they prefer.

**Comment by jerle76** [comment]
fair enough! I should have the simplified CreateJS version ready tonight.
then I will implement the level selection with L/R arrows which is missing in CreateJS

**Comment by jerle76** [comment]
turns out the CreateJS version was always using the prerendered background... that made things simpler! I'm just left with the level selection that I'll do another night

**Comment by jerle76** [comment]
And I've finished the level selection with L/R arrow keys. I'll leave the CreateJS version alone now ;p
